main.go

```diff
@@ -2297,6 +2303,119 @@ The following options are recognized:
 		return nil
 	})
 
+	r.Dispatch("x509 renew", &Help{
+		Summary: "Renew X.509 Certificates and Certificate Authorities",
+		Usage:   "safe x509 renew [OPTIONS] path/to/certificate",
+		Description: `
+Renew an X.509 Certificate
+
+The following options are recognized:
+
+  -i, --signed-by   Path in the Vault where the CA certificate
+                    (and signing key) can be found.  If this is not
+                    provided, a sibling secret named 'ca' will used
+                    if it exists.
+
+  -t, --ttl         How long the new certificate will be valid
+                    for.  Specified in units h (hours), m (months)
+                    d (days) or y (years).  1m = 30d and 1y = 365d
+                    Defaults to same duration used by the certificate
+                    starting at the time of the renew.
```

Comment:
"starting at the time of the renew" is confusing here.  I think you can rephrase as "Defaults to the original TTL used to issue the certificate"

Reply:
k

Reply:
used `Defaults to the last TTL used to issue or renew the certificate` as that's more accurate.